src/lib/utils/hooks/SynapseAPI/forum/useForum.ts

```diff
@@ -54,18 +61,128 @@ export function useGetForumInfinite(
   )
 }
 
-export function useGetThread(
+export function useGetThread(threadId: string) {
+  const { data: threadData, isLoading: isLoadingBundle } =
+    useGetThreadBundle(threadId)
+  const { data: threadBody, isLoading: isLoadingBody } = useGetThreadBody(
+    threadData?.messageKey ?? '',
```

Comment:
The parameter for this hook requires the response from the above hook. However, when first instantiated it runs with an empty parameter throwing a 400 error. The expected response is eventually returned but is there any other suggestions?

Reply:
Does this fix it?

```suggestion
  const { data: threadBody, isLoading: isLoadingBody } = useGetThreadBody(
    threadData?.messageKey ?? '',
    { enabled: !!(threadData?.messageKey) }
```